Mlem/Views/Tabs/Settings/Components/Views/About/AboutView.swift

```diff
@@ -0,0 +1,118 @@
+                                print("Couldn't load Mlem community")
+                            }
+                        }
+                        
+                    } label: {
+                        Label("Official Community", systemImage: "house.fill").labelStyle(SquircleLabelStyle(color: .green, fontSize: 15))
+                    }
+                    .buttonStyle(.plain)
+                    
+                    Link(destination: URL(string: "https://matrix.to/#/%23mlemapp:matrix.org")!) {
+                        Label("Matrix Room", systemImage: "chart.bar.doc.horizontal").labelStyle(SquircleLabelStyle(color: .teal))
+                    }
+                    .buttonStyle(.plain)
+                }
+                
+                Section {
+                    Link(destination: URL(string: "https://github.com/mlemgroup/mlem")!) {
+                        Label("Github Repository", image: "logo.github").labelStyle(SquircleLabelStyle(color: .black))
+                    }
+                    .buttonStyle(.plain)
+                    
+                    NavigationLink {
+                        ContributorsView()
+                    } label: {
+                        Label("Contributors", systemImage: "person.2.fill").labelStyle(SquircleLabelStyle(color: .teal))
+                    }
+                }
+                
+                Section {
+                    NavigationLink {
+                        ScrollView {
+                            MarkdownView(text: privacyPolicy.body, isNsfw: false)
+                                .padding()
+                        }
+                    } label: {
+                        Label("Privacy Policy", systemImage: "hand.raised.fill").labelStyle(SquircleLabelStyle(color: .blue))
+                    }
+                    NavigationLink {
+                        ScrollView {
+                            MarkdownView(text: eula.body, isNsfw: false)
+                                .padding()
+                        }
+                    } label: {
+                        Label("EULA", systemImage: "doc.plaintext.fill").labelStyle(SquircleLabelStyle(color: .purple))
+                    }
+                    NavigationLink {
+                        LicensesView()
+                    } label: {
+                        Label("Licenses", systemImage: "doc.fill").labelStyle(SquircleLabelStyle(color: .orange))
+                    }
+                }
+            }
+        }
+        .navigationTitle("About")
+    }
+    
+    var versionString: String {
+        var result = "n/a"
+
+        if let releaseVersion = Bundle.main.releaseVersionNumber {
+            result = releaseVersion
+        }
+
+        if let buildVersion = Bundle.main.buildVersionNumber {
+            result.append(" (\(buildVersion))")
+        }
+
+        return result
+    }
+    
+    @ViewBuilder
+    private var appHeaderView: some View {
+        VStack(spacing: 10) {
```

Comment:
10 should be `AppConstants.postAndCommentSpacing` (that's our default spacing--the name is long overdue for a refactor lol)